octopod-backend/src/Octopod/Server/Args.hs

```diff
@@ -26,6 +26,10 @@ data Args = Args
     db :: ByteString
   , -- | database pool size
     dbPoolSize :: Int
+  , -- | log debug messages
+    debug :: Bool
+  , -- | whether log output should be minimal
+    minimal :: Bool
```

Comment:
I propose we define a logging format explicitly for local development (maybe even call it `local-dev`?)